frontend/projects/valtimo/admin-settings/src/lib/components/admin-settings-color/admin-settings-color.component.html

```
@@ -0,0 +1,90 @@
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<form
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)